src/style.css

```diff
@@ -2,0 +1,73 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir-Book, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+}
```

Comment:
Be consistent with your margins. Add only top or bottom margins, but not both, to avoid potential margin collapse issues.